homeworks/hw12/src/bank.rs

```diff
@@ -0,0 +1,869 @@
+use log::{debug, error, info};
+/// This module contains the implementation of a simple banking system.
+///
+/// The `Bank` struct represents a bank and provides methods for managing accounts
+/// and performing various banking operations such as deposits, withdrawals, and transfers.
+///
+use serde::{Deserialize, Serialize};
+use std::cell::RefCell;
+use std::collections::{BTreeMap, HashMap};
+use std::{thread, time};
+use thiserror::Error;
+use ulid::Ulid;
+
+type Money = f64;
+
+const MONEY_ZERO: Money = 0.0;
+
+type TransactionId = String;
+
+#[derive(Default)]
+pub struct Bank {
+    accounts: HashMap<String, RefCell<Money>>,
+    accounts_history: HashMap<String, Vec<String>>,
```

Comment:
```suggestion
    accounts_history: HashMap<String, Vec<TransactionId>>,
```

Думаю, подразумевалось вот это)